### ISSUE TITLE
Anti-epidermal transglutaminase (eTG) antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6605]
**Preferred term label:** Anti-epidermal transglutaminase (eTG) antibody positivity

**Synonyms**
Anti-keratinocyte transglutaminase antibody positivity

**Definition (free text, please give PubMed ID)**
Antibodies to epidermal transglutaminase (eTG, also keratinocyte transglutaminase).

**Parent term (use hpo.jax.org/app)**
Anti-transglutaminase antibody positivity (once created)

**Diseases characterized by this term  (e.g. Orphanet or OMIM number)**
Dermatitis herpetiformis ORPHA:1656

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
